Create 20250221.md (#212)
ESMValGroup/Community: User Engagement Lead/Minutes/20250221.md

## UET Meeting Feb 21st 2025

### Attendees 
@rswamina, @hb326, @ehogan

### Meeting Notes

> 1. Updating ESMValTool tutorial for the current planned release of ESMValTool (v1.12):

Reviewer: ```suggestion
1. Updating ESMValTool tutorial for the current planned release of ESMValTool (v2.12):
```

   1. Plan ahead to make updates by end of May 2025.
   2. @rswamina to create two issues - one to list essential updates and other for more detailed or aspirational updates incluing new topics.
   3. UET to identify what they will be updating and who will be doing the different updates before the workshop.
   4. Dedicated sessions for actually making the updates at May workshop.
2. More general tutorial updates
   1. Tag tutorial for version compatibility with new ESMValTool releases.
   2. Divide tutorial into two repositories - one basic that gets updated regularly and two advanced that is more community driven and may not be updated as regularly. @ehogan to get feedback from tech lead team for this and also to be discussed in the next community meeting.
   3. Switching to new software carpentry template - @ehogan to get feedback from tech lead team for this as well.

> 3. There is an ESMValTool splinter meeting at EGU in April  @hb326 to check if online participation is possible.

Reviewer: ```suggestion
3. There is an ESMValTool splinter meeting at EGU in April - @hb326 to check if online participation is possible.
```

4. Community engagement:
   1. Discussed having one community wide training event and ESMValTool surgeries through the year.
   2. For the community event @hb326 to check with contact from Sweden who requested a tutorial, perhaps something that can be done with SMHI.

>    3. Plan to convert one or more community monthly meetings into ESMValTool surgeries where people can ask 1-1 questions. We will ensure there are participants from the Tech Lead and UET teamsat those sessions and advertise well before the meeting.

Reviewer: ```suggestion
   3. Plan to convert one or more community monthly meetings into ESMValTool surgeries where people can ask 1-1 questions. We will ensure there are participants from the Tech Lead Team and UET at those sessions and advertise well before the meeting.
```

Author: @ehogan - Happy with changes. Please go ahead and commit and merge. Thanks!
